CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ## Added
 - Util to compare value types
+- Add v-tooltip dependency
```

Comment:
That's very detailed!  It's fine if the changelog has just one entry by PR, it's good to keep it concise so it's easy to read.